tests/test_pylast.py

```diff
@@ -23,6 +23,7 @@ def load_secrets():
         try:
             doc["username"] = os.environ['PYLAST_USERNAME'].strip()
             doc["password_hash"] = os.environ['PYLAST_PASSWORD_HASH'].strip()
+            doc["token"] = os.environ['PYLAST_TOKEN'].strip()
```

Comment:
What should be put in this environment variable? Because it's missing, all the integration tests are skipped.